Libraries/Text/Text/RCTTextView.m

```diff
@@ -119,14 +120,19 @@ - (void)setSelectable:(BOOL)selectable
 
   _selectable = selectable;
 
-#if !TARGET_OS_OSX // TODO(macOS GH#774)
+#if !TARGET_OS_OSX // [TODO(macOS GH#774)
   if (_selectable) {
     [self enableContextMenu];
   }
   else {
     [self disableContextMenu];
   }
-#endif // TODO(macOS GH#774)
+#else
+  _textView.selectable = _selectable;
+  if (_selectable) {
+    [self setFocusable:YES];
```

Comment:
Keep previous behavior where `selectable={true}` will be `focusable` unless disabled.